### PR TITLE
Remove API Blueprint AST from public API

### PR DESCRIPTION
**Pull request by klokane**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### Breaking
 
+* Drop support for Blueprint AST (in API)
```
Comment (pksunkara) [inline CHANGELOG.md]: `- Removed the option to select AST Type. The ouput will be only refract`

**Comment by pksunkara** [comment]
LGTM. Only a small comment.

**Comment by klokane** [comment]
There is still API Blueprint AST in "performance", working on fix

**Comment by klokane** [comment]
Damn, do I missing something, or we really assign into `optionsResult->error` value of string allocated just on stack?

There:
https://github.com/apiaryio/protagonist/blob/master/src/options_parser.cc#L60
https://github.com/apiaryio/protagonist/blob/master/src/options_parser.cc#L71
https://github.com/apiaryio/protagonist/blob/master/src/options_parser.cc#L81

**Comment by klokane** [comment]
@w-vi @kylef @pksunkara please see above

**Comment by pksunkara** [comment]
I will leave it completely upto you.

**Comment by klokane** [comment]
Fixed, @w-vi please check for memory alloc/deallloc

**Comment by richthegeek** [comment]
What was the motivation behind this change? It seems to remove a feature for no real benefit. Most of the reduced code was in the tests?

**Comment by pksunkara** [comment]
@richthegeek We have deprecated API Blueprint AST long ago and with the next major release will be completely removing it. I wouldn't consider it a feature. We have been using [API Elements](http://api-elements.readthedocs.io) for a long time.

**Pull request merged**